src/impy/models/dpmjetIII.py

```diff
@@ -78,24 +78,35 @@ def __init__(self, event_kinematics, seed=None, logfname=None):
         self._max_A2 = event_kinematics.A2
 
         self.event_kinematics = event_kinematics
-        dpm_conf = impy_config["dpmjetIII"]
 
         info(1, "Initializing DPMJET-III")
+
+        if self.version == "3.0-6":
+            dat_dir_name = "dpm3"
+            param_file_name = "fitpar.dat"
+            evap_file_name = "dpmjet.dat"
+        else:
+            dat_dir_name = "dpm3191"
+            param_file_name = "dpmjpar.dat"
+            evap_file_name = "dpmjet.dat"
+
+        data_dir = _cached_data_dir(dat_dir_name)
+
         # Set the dpmjpar.dat file
         if hasattr(self._lib, "pomdls") and hasattr(self._lib.pomdls, "parfn"):
-            pfile = dpm_conf["param_file"][self.version]
+            pfile = data_dir + param_file_name
```

Comment:
Shouldn't that be a path object or so?? Now this totally depends if there is a backslash or not

Reply:
Fortran code requires "/" in the end of directory.